legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/auth/GitLabSession.java

```diff
@@ -28,11 +28,13 @@
 
     boolean gitLabOAuthCallback(GitLabMode mode, String code);
 
-    String getAccessToken(GitLabMode mode);
+    GitLabToken getGitLabToken(GitLabMode mode);
 
-    void clearAccessTokens();
+    void clearGitLabTokens();
 
-    void putAccessToken(GitLabMode mode, String token);
+    void putGitLabToken(GitLabMode mode, String token);
```

Comment:
Do we need this interface? It seems like it's underspecified, since it doesn't take the token type.